src/components/DateRangePicker.jsx

```diff
@@ -43,6 +43,7 @@ const defaultProps = {
   numberOfMonths: 2,
   showClearDates: false,
   disabled: false,
+  hidePickerOnClear: false,
```

Comment:
I think that would be fine. We can couple the release with https://github.com/airbnb/react-dates/pull/53 for 3.0.0. I think, semantically, `reopenPickerOnClearDates` seems like a better name